src/newc.rs

```diff
@@ -66,6 +66,32 @@ fn pad(len: usize) -> Option<Vec<u8>> {
     }
 }
 
+pub enum ModeFlag {
+    Symlink,
+    Fifo,
+    Char,
+    Block,
+    NetworkSpecial,
+    Socket,
+    Directory,
+    Regular,
+}
+
+impl From<ModeFlag> for u32 {
+    fn from(m: ModeFlag) -> u32 {
+        match m {
+            ModeFlag::Fifo => 0o010000,
+            ModeFlag::Char => 0o020000,
+            ModeFlag::Directory => 0o040000,
+            ModeFlag::Block => 0o060000,
+            ModeFlag::Regular => 0o100000,
+            ModeFlag::NetworkSpecial => 0o110000,
```

Comment:
Where are you getting this value from? I see all of the other values in the `cpio(5)` manpage, but I don't see the `NetworkSpecial` value.

Reply:
From cpio's source